### PR TITLE
DDCE-3011 Update select client content

### PR DESCRIPTION
**Pull request by mywyau**
# DDCE-3011

Welsh content update to match translation from welsh team.

https://jira.tools.tax.service.gov.uk/browse/DDCE-3011

ATs --> https://github.com/hmrc/tax-history-acceptance-tests/pull/158

[DDCE 3004-010-011-012-013 - Welsh Translation .xlsx](https://github.com/hmrc/tax-history-frontend/files/9878555/DDCE.3004-010-011-012-013.-.Welsh.Translation.xlsx)

## Checklist PR Raiser
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date
 
## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by mywyau** [comment]
Screenshot 2022-10-27 at 11 26 33
Screenshot 2022-10-27 at 11 26 42

**Comment by mywyau** [comment]
[DDCE 3004-010-011-012-013 - Welsh Translation .xlsx](https://github.com/hmrc/tax-history-frontend/files/9878555/DDCE.3004-010-011-012-013.-.Welsh.Translation.xlsx)

**Comment by mywyau** [comment]
ATs -->. https://github.com/hmrc/tax-history-acceptance-tests/pull/158

File: conf/messages.en [hunk 1]
```diff
@@ -151,8 +151,8 @@ tax.code.total.deductions=Total deductions
 
 ################################ Select Client ####################################################
 employmenthistory.select.client.continue=Continue
-employmenthistory.select.client.title=Enter a client''s National Insurance number
-employmenthistory.select.client.heading=Enter a client''s National Insurance number
```
Comment (mywyau) [inline conf/messages.en:155]: Removing double apostrophes and using the correct one

File: test/views/taxhistory/select_clientSpec.scala [hunk 2]
```diff
@@ -33,38 +33,39 @@ import views.{BaseViewSpec, Fixture}
...
-    val validForm: Form[SelectClient]              = selectClientForm.bind(postData, maxChars)
+    lazy val nino: String = randomNino.toString()
+    val postData: JsObject = Json.obj("clientId" -> nino)
+    val validForm: Form[SelectClient] = selectClientForm.bind(postData, maxChars)
     val invalidFormWrongFormat: Form[SelectClient] =
       selectClientForm.bind(Json.obj("clientId" -> "123456#$&"), maxChars)
   }
 
+  val titleAndHeadingContent = "Enter your client’s National Insurance number"
+
```
Comment (mywyau) [inline test/views/taxhistory/select_clientSpec.scala:47]: Changed how the unit test for this page checks tests the content. The old method of testing the heading and title would always pass regardless of what I added or changed in the message conf files.

File: conf/messages.cy [hunk 3]
```diff
@@ -218,11 +218,11 @@ employmenthistory.mci.restricted.text=I gael rhagor o wybodaeth, cysylltwch â C
 employmenthistory.mci.restricted.telephone=dros y ffôn ar 0300 200 1900
 employmenthistory.mci.restricted.textphone=dros ffôn testun (Saesneg) ar 0300 200 3319
 employmenthistory.mci.restricted.title=Nid oes gennych fynediad at gofnod incwm y cleient hwn
-employmenthistory.mci.restricted.select.client.link.text=Nodwch rif Yswiriant Gwladol cleient arall
+employmenthistory.mci.restricted.select.client.link.text=Nodwch rif Yswiriant Gwladol eich cleient arall
```
Comment (Taiwo-AB) [inline conf/messages.cy:221]: Please revert this. The previous one is the correct one.

File: conf/messages.cy [hunk 4]
```diff
@@ -218,11 +218,11 @@ employmenthistory.mci.restricted.text=I gael rhagor o wybodaeth, cysylltwch â C
...
 employmenthistory.deceased.header=Nid oes gennych fynediad at gofnod incwm y cleient hwn
 employmenthistory.deceased.text=Gwiriwch gyda’ch cleient.
 employmenthistory.deceased.title=Nid oes gennych fynediad at gofnod incwm y cleient hwn
-employmenthistory.deceased.select.client.link.text=Nodwch rif Yswiriant Gwladol cleient arall
+employmenthistory.deceased.select.client.link.text=Nodwch rif Yswiriant Gwladol eich cleient arall
```
Comment (Taiwo-AB) [inline conf/messages.cy:225]: Please revert this. The previous one is the correct one.

File: conf/messages.en [hunk 5]
```diff
@@ -151,8 +151,8 @@ tax.code.total.deductions=Total deductions
...
+employmenthistory.select.client.title=Enter your client’s National Insurance number
```
Comment (Taiwo-AB) [inline conf/messages.en:154]: Due to the apostrophe changes, its rendering does not match the prototype which was the case previously. Please revert.

File: conf/messages.en [hunk 6]
```diff
@@ -151,8 +151,8 @@ tax.code.total.deductions=Total deductions
...
+employmenthistory.select.client.heading=Enter your client’s National Insurance number
```
Comment (Taiwo-AB) [inline conf/messages.en:155]: Due to the apostrophe changes, its rendering does not match the prototype which was the case previously. Please revert.

File: project/build.properties [hunk 7]
```diff
@@ -1 +1 @@
-sbt.version=1.6.2
+sbt.version=1.7.1
```
Comment (Taiwo-AB) [inline project/build.properties:1]: Can you please bump this to 1.7.2? I would do the other dependency bumps or cleanup as part of ticket I'm working on. Thanks

**Comment by mywyau** [comment]
Screenshot 2022-10-27 at 14 59 50

File: conf/messages.cy (hunk 3 above)
Comment (Taiwo-AB) [reply conf/messages.cy:221]: This is not part of what needs changing. This is a mci restricted scenario which is one of the client error pages where you are asked to `Enter another client's National Insurance number` due to failure.

**Review by Taiwo-AB — APPROVED** [review]

**Pull request merged**